server/socket/socketApi.js

```diff
@@ -0,0 +1,24 @@
+const io = require("socket.io")();
+const socketApi = { io };
+
+socketApi.sockets = [];
```

Comment:
This should probably be an Object, so you don't need to use indexOf all the time (O(n) solution)